src/sh/01-led-light.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+echo LED Indicator
+mount -t sysfs sysfs /sys
+echo heartbeat > /sys/class/leds/led1/trigger 
+echo none > /sys/class/leds/led0/trigger
+echo 0 > /sys/class/leds/led0/brightness
+
+cat <<"EOF"> /usr/local/bin/bootedled
+#!/bin/sh
+echo 1 > /sys/class/leds/led1/brightness 
+echo 1 > /sys/class/leds/led0/brightness
+exit 0 
+EOF
+chmod +x /usr/local/bin/bootedled
+
+cat <<"EOF"> /lib/systemd/system/bootedled.service
+[Unit]
+Description=LED Boot
+Wants=multi-user.target
```

Comment:
@darkdrgn2k I am not familiar with systemd, but isn't this a loop when it both `Wants` and is `WantedBy` the `multi-user.target`? Which way is the dependency?

Reply:
I'm  it 100% sure. I got this from a site trying to get a service that starts last. (since rc.local does not exist)

Wants is not require maybe that's why it's not a loop?